lib/twurl/cli.rb

```diff
@@ -92,15 +90,15 @@ def parse_options(args)
           end
         end
 
-        arguments                 = option_parser.parse!(args)
-        Twurl.options.command     = extract_command!(arguments)
-        Twurl.options.path        = extract_path!(arguments)
-
-        if Twurl.options.command == DEFAULT_COMMAND and Twurl.options.path.nil?
+        if args.first.nil?
```

Comment:
```suggestion
        if Twurl.options.command == DEFAULT_COMMAND and args.first.nil?
```

Any specific reason you removed the first condition here?

Reply:
I think we should keep my change. That way twurl can show command usage when someone invokes it without any arguments.

Reply:
@flaviomartins 
https://github.com/twitter/twurl/blob/master/lib/twurl/cli.rb#L139-L145

`$ twurl` should return usage output regardless of this change. Can you confirm?

Reply:
In that case, I added some extra conditions. We can only raise it when the program is run without arguments and if it is not an alias.